pointerevents/pointerevent_capture_suppressing_mouse-manual.html

```diff
@@ -18,10 +18,11 @@
                 <li> Move your mouse out of the black rectangle. pointerout and pointerleave should be logged inside of it</li>
                 <li> Put your mouse over the purple rectangle. pointerover and pointerenter should be logged inside of it.</li>
                 <li> Move your mouse out of the purple rectangle. pointerout and pointerleave should be logged inside of it</li>
-                <li> Press and hold left mouse button over "Set Capture" button. "gotpointercapture" should be logged in the black rectangle</li>
+                <li> Press and hold left mouse button over "Set Capture" button and move mouse a litte inside the button. "gotpointercapture", "pointerover", and "pointerenter" should be logged in the black rectangle</li>
                 <li> Put your mouse over the purple rectangle and then move it out. Nothing should happen</li>
-                <li> Put your mouse over the black rectangle. pointerover and pointerenter should be logged inside of it.</li>
-                <li> Move your mouse out of the black rectangle. pointerout and pointerleave should be logged inside of it</li>
+                <li> Put your mouse over the black rectangle and then move it out. Nothing should happen.</li>
+                <li> Put your mouse over the purple rectangle and then release left mouse button. "lostpointercapture" should be logged in the black rectangle and pointerover</li>
+                <li> Move your mouse in the purple rectangle a little. "pointerout" and "pointerleave" should be logged in the black rectangle. Also "pointerover" and "pointerenter" should be logged in the purple rectangle"</li>
```

Comment:
Combining this step with above makes it easier. The event description would become too long but it doesn't matter, right?